### PR TITLE
Updating the dependencies to the latest version

### PR DESCRIPTION
**Pull request by petjofi**
Hello Tim Branyen,

Thank you for the awesome repos you shared with us. Not just this boilerplate, but all. I just wanted to let you know that I managed to update the dependencies in this one to the latest versions. If you are interested you can merge my work. I'm trying to make a reusable Backbone-Ractive boilerplate and I hope you don't mind me using your repo as a base. I understand you're probably very busy with Netflix but your feedback is highly appreciated. Thank you and good luck.

Cheers,
petjofi

### CODE REVIEW

File: test/tests/example.js [hunk 1]
```diff
@@ -1,46 +1,52 @@
-describe("Simple tests examples", function() {
-  it("should detect true", function() {
+describe("Simple tests examples", function () {
```
Comment (tbranyen) [inline test/tests/example.js]: Noticed a style deviation here. Converted from `function() {` to `function () {`. Unfortunately no style/lint rule caught this, sorry about that.

**Comment by tbranyen** [comment]
Looks good to me, thanks @petjofi! I left one comment about a style deviation, other than that :+1:

File: test/tests/example.js (hunk 1 above)
Comment (petjofi) [inline test/tests/example.js]: Thank you for your feedback. Actually it is my fault. I just wanted to test if the karma update was successful. I shouldn't commit the change on the example tests at all. I will roll back the changes on this file.

**Comment by tbranyen** [comment]
Looks good, thanks for the PR @petjofi!

**Pull request merged**